clash-lib/clash-lib.cabal

```diff
@@ -279,10 +279,10 @@ Library
 
                       Clash.Verification.Pretty
 
+                      Data.Aeson.Extra
```

Comment:
I know we don't do this for `Data.Text.Prettyprint.Doc.Extra`, but if you're moving a `*.Extra` module from `other-modules` to `exposed-modules` I think it should have a comment why it's there (because generally speaking I disagree with such modules being part of the public API unless absolutely necessary)